Generator/Product/ProductRawBuilder.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+
+namespace Pim\Bundle\DataGeneratorBundle\Generator\Product;
+
+use Akeneo\Component\Classification\Repository\CategoryRepositoryInterface;
+use Faker;
+use Pim\Bundle\CatalogBundle\Model\AttributeInterface;
+use Pim\Bundle\CatalogBundle\Model\FamilyInterface;
+use Pim\Bundle\CatalogBundle\Repository\AttributeRepositoryInterface;
+
+/**
+ * Build a raw product (ie: as an array) with random data.
+ *
+ * @author    Julien Janvier <jjanvier@akeneo.com>
+ * @copyright 2014 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ProductRawBuilder
+{
+    const CATEGORY_FIELD = 'categories';
+
+    /** @var Faker\Generator */
+    private $faker;
+
+    /** @var ProductValueRawBuilder */
+    private $valueRawBuilder;
+
+    /** @var AttributeRepositoryInterface */
+    private $attributeRepository;
+
+    /** @var CategoryRepositoryInterface */
+    private $categoryRepository;
+
+    /** @var array */
+    private $attributesByFamily;
+
+    /** @var array */
+    private $categoryCodes;
+
+    /** @var string */
+    private $identifierCode;
```

Comment:
why? such tools are a good field to test good things that we can't/don't do in the PIM